### PR TITLE
fix mypy failures in newly merged code

### PR DESCRIPTION
**Pull request by manasghandat**
fixes mypy v1.1.1 failures in code newly merged around the same time as mypy v1.1.1 was merged.

### Checklist

- [x] No CHANGELOG update needed

- [x] No new tests needed

- [x] No documentation update needed

### CODE REVIEW

File: capa/render/proto/__init__.py [hunk 1]
```diff
@@ -502,27 +502,36 @@ def metadata_from_pb2(meta: capa_pb2.Metadata) -> rd.Metadata:
...
+                functions=tuple(
+                    [
+                        rd.FunctionLayout(
+                            address=addr_from_pb2(f.address),
+                            matched_basic_blocks=tuple(
+                                [
+                                    rd.BasicBlockLayout(address=addr_from_pb2(bb.address))
+                                    for bb in f.matched_basic_blocks
+                                ]
+                            ),
```
Comment (williballenthin) [inline capa/render/proto/__init__.py:514]: thanks for making these changes

File: capa/render/proto/__init__.py [hunk 2]
```diff
@@ -585,13 +594,13 @@ def feature_from_pb2(f: capa_pb2.FeatureNode) -> frzf.Feature:
         return frzf.ExportFeature(export=ff.export, description=ff.description or None)
     elif type_ == "import_":
         ff = f.import_
-        return frzf.ImportFeature(import_=ff.import_, description=ff.description or None)
+        return frzf.ImportFeature(import_=ff.import_, description=ff.description or None)  # type: ignore
```
Comment (williballenthin) [inline capa/render/proto/__init__.py:597]: please add comments describing why we're overriding mypy

File: capa/render/proto/__init__.py [hunk 3]
```diff
@@ -694,7 +703,8 @@ def maec_from_pb2(pb: capa_pb2.MaecMetadata) -> rd.MaecMetadata:
         malware_family=pb.malware_family or None,
         malware_category=pb.malware_category or None,
         malware_category_ov=pb.malware_category_ov or None,
-    )
+    )  # type: ignore
+    # Mypy is unable to recognise arguments due to alias
```
Comment (williballenthin) [inline capa/render/proto/__init__.py:707]: these comments are good

File: capa/render/proto/capa_pb2.py
Comment (williballenthin) [inline capa/render/proto/capa_pb2.py:1]: please revert changes to these autogenerated files (`*_py2.*`). they are created by the protobuf compiler, not by hand, so we should try to keep them pristine.

**Review by williballenthin — CHANGES_REQUESTED** [review]
few requests inline

**Comment by mr-tz** [comment]
what happened with the close here?

**Comment by williballenthin** [comment]
no, i must have hit the close button by mistake 🤦🏼 
and since the branch got deleted i'll have to recreate the changes and attribute them to @manasghandat 
sorry about that!

**Comment by mr-tz** [comment]
oh no, @manasghandat if you still have the changes handy please open a new PR

**Comment by manasghandat** [comment]
@mr-tz @williballenthin Can you please create a branch so that I can commit to or should I commit to the main branch?

**Comment by williballenthin** [comment]
Sorry for the mistake here. I’ll reconstruct the code proposed by @manasghandat and find a way to reopen this PR or create a new one.

**Comment by manasghandat** [comment]
@williballenthin  I do have to code and I have made the recommended changes too. It will be great if you could just make a branch for me to commit to.

**Comment by williballenthin** [comment]
please see the new PR in #1412 and continue work there.  i think outstanding are a few mypy override comments and reverting changes to autogenerated files.

**Pull request closed without merging**